package chapter12_exception_handling_and_text_io.exercise;

public class Test
{
    public static void main(String[] args)
    {

    }
}
